### PR TITLE
Release v0.7.0

### PR DESCRIPTION
**Pull request by z4y4ts**
### Added

- XBlock SDK Workbench scenarios to simplify development.
- Acceptance tests on top of XBlock SDK Workbench.
- Usage documentation.

### Removed

- Removed TravisCI integration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,40 @@ settings:
 
 ## Usage
 
-TODO
+To embed a video simply copy & paste it's URL into a `Video URL` field.
```
Comment (bryanlandia) [inline README.md:53]: tiny thing.  should be "its" not "it's"

File: README.md [hunk 2]
```diff
@@ -50,7 +50,40 @@ settings:
...
+
+- Brightcove: https://studio.brightcove.com/products/videocloud/media/videos/12345678 or https://studio.brightcove.com/products/videos/12345678
+- HTML5: https://example.com/video.mpeg|mp4|ogg|webm
+- Vimeo: https://vimeo.com/abcd1234
+- Wistia: https://raccoongang.wistia.com/medias/xmpqebzsya
+- Youtube: https://www.youtube.com/watch?v=3_yD_cEKoCk and others.
+
+### Brightcove
+
+To successfully use videos hosted on Brightcove Videocloud service one must provide valid Brightcove `account_id` associated with the video. One can find
```
Comment (bryanlandia) [inline README.md:65]: Looks like this is an unfinished thought.

File: README.md [hunk 3]
```diff
@@ -50,7 +50,40 @@ settings:
...
+
+#### Connect to Brightcove Platform
+
+1. Grab your [BC_TOKEN] from Brightcove Videocloud.
+1. Open Video XBlock settings, Advanced tab. Scroll down to `Video API Token` section.
+1. Put `BC_TOKENT` taken from Brightcvove into `Client Token` field.
```
Comment (bryanlandia) [inline README.md:71]: `BC_TOKEN`

**Review by bryanlandia — APPROVED** [review]
Will we continue to need the RacoonGang forks and branches of  videojs dependencies?  Would you be able to summarize what the changes are in your forks of videojs deps vs. the main repo, and if these are changes which are suggested for upstream PR to videojs, or particular to use in the XBlock?

**Comment by z4y4ts** [comment]
@bryanlandia re: videojs plugins forks. I'm working on integrating those changes back into upstream repos. Mainly those are small fixes, like incomplete implementation of videojs-plugin interface or a bug which caused "double clicking" triggering on a play buggon.
It's going to take some time though.

File: README.md (hunk 2 above)
Comment (z4y4ts) [reply README.md:65]: Yeah, sorry, missed that. Will submit another PR with the complete sentense :)

**Pull request merged**